doc/usage/installation.rst

```diff
@@ -0,0 +1,147 @@
+=================
+Installing Sphinx
+=================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+
+Linux
+-----
+
+Debian/Ubuntu
+~~~~~~~~~~~~~
+
+Install ``python-sphinx`` using :command:`apt-get`:
+
+.. code-block:: bash
+
+   $ apt-get install python-sphinx
+
+If it not already present, this will install Python for you.
+
+RHEL, CentOS
+~~~~~~~~~~~~
+
+Install ``python-sphinx`` using :command:`yum`:
+
+.. code-block:: bash
+
+   $ yum install python-sphinx
+
+If it not already present, this will install Python for you.
+
+Other distributions
+~~~~~~~~~~~~~~~~~~~
+
+Most Linux distributions have Sphinx in their package repositories.  Usually
+the package is called ``python-sphinx`` or ``sphinx``.  Be aware that there are
+at least two other packages with ``sphinx`` in their name: a speech recognition
+toolkit (*CMU Sphinx*) and a full-text search database (*Sphinx search*).
+
+
+macOS
+-----
+
+Sphinx can be installed using `MacPorts <http://www.macports.org/>`_.
+
+.. code-block:: bash
+
+   $ sudo port install py27-sphinx
+
+To set up the executable paths, use the ``port select`` command:
+
+.. code-block:: bash
+
+   $ sudo port select --set python python27
+   $ sudo port select --set sphinx py27-sphinx
+
+Type :command:`which sphinx-quickstart` to check if the installation was
+successful.
+
+
+Windows
+-------
+
+.. todo:: Could we start packaging this?
+
+Most Windows users do not have Python installed by default, so we begin with
+the installation of Python itself.  If you are unsure, open the *Command
+Prompt* (:kbd:`Control-r` and type :command:`cmd`).  Once the command prompt is
+open, type :command:`python --version` and press Enter.  If Python is
+available, you will see the version of Python printed to the screen.  If you do
+not have Python installed, refer to the *Hitchhikers Guide to Python*\'s Python
+on Windows installation guides. You can install either `Python 3`__ `Python
+2.7`__ Python 3 is recommended.
+
+Once Python is installed, you can install Sphinx using :command:`pip`.  Refer
+to the :ref:`pip installation instructions <install-pypi>` below for more
+information.
+
+__ http://docs.python-guide.org/en/latest/starting/install3/win/
+__ http://docs.python-guide.org/en/latest/starting/install/win/
+
+
+.. _install-pypi:
+
+Installation from PyPI
+----------------------
+
+Sphinx packages are published on the `Python Package Index
+<https://pypi.python.org/pypi/Sphinx>`_.  The preferred tool for installing
+packages from *PyPI* is :command:`pip`.  This tool is provided with all modern
+versions of Python.
+
+On Linux or MacOS, you should open your terminal and run the following command.
+
+.. code-block:: shell
+
+   $ pip install -U sphinx
+
+On Windows, you should open *Command Prompt* (:kbd:`Control-r` and type
+:command:`cmd`) and run the same command.
+
+.. code-block:: bat
+
+   C:\> pip install -U sphinx
+
+After installation, type :command:`sphinx-build -h` on the command prompt.  If
```

Comment:
I prefer `sphinx-build --version`. But either is okay.

Reply:
Done